<!-- Launch file for the rrt-detector -->

<launch>
    <arg name="eta" value="10.0"/>
    <arg name="Geta" value="15.0"/>

    <node pkg="robosar_task_allocator" type="frontier_detector_node" name="frontier_detector_node" output="screen">
        <param name="eta" value="$(arg Geta)"/>
        <param name="map_topic" value="/map"/>
    </node>

     <node pkg="robosar_task_allocator" type="frontier_filter.py" name="frontier_filter" output="screen">
        <param name="map_topic" value="/map"/>
        <param name="info_radius" value="0.5"/> 
        <param name="info_threshold" value="0.2"/> 
        <param name="costmap_clearing_threshold" value="70"/> <param name="costmap_clearing_threshold" value="70"/> 
        <param name="cluster_bandwidth" value="1.0"/> 
        <param name="goals_topic" value="/detected_points"/>
        <param name="rate" value="1"/>
    </node>

</launch>